src/execution/Execute.tsx

```diff
@@ -118,7 +119,7 @@ export class Execute extends React.Component<IExecuteProps, IExecuteState> {
     }
 
     private runExecution(props: IExecuteProps) {
-        const { afm, resultSpec, onLoadingChanged } = props;
+        const { afm, onLoadingChanged } = props;
```

Comment:
please keep resultSpec here, and rename `resultSpec` variable below (don't use `this.props.resultSpec` in this function when `props` parameter is `this.props`)

Reply:
done